### PR TITLE
Add check HTML conversion

### PR DESCRIPTION
**Pull request by askonev**

### CODE REVIEW

File: spec/convertion_by_format/check_open_html_by_screen_spec.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require_relative '../spec_helper'
```
Comment (ShockwaveNN) [inline spec/convertion_by_format/check_open_html_by_screen_spec.rb:3]: require 'spec_helper` should work fine I think

**Review by ShockwaveNN — CHANGES_REQUESTED** [review]

**Review by ShockwaveNN — APPROVED** [review]

File: spec/convertion_by_format/check_open_html_by_screen_spec.rb (hunk 1 above)
Comment (askonev) [reply spec/convertion_by_format/check_open_html_by_screen_spec.rb]: Yes, you right, but ruby mine is not found in the declaration 

image

### Rubymine linter rule

Reports references in code which can't be resolved to any declaration. Some example references would be a method reference, or a constant reference. Unresolved references are often caused by typos, and can lead to NoMethodErrors, NameErrors and more at runtime.

Comment (ShockwaveNN) [reply spec/convertion_by_format/check_open_html_by_screen_spec.rb]: This is known rubymine's linter [bug](https://youtrack.jetbrains.com/issue/RUBY-25223)

Running this file should be ok, especially as other files in this project has same `require 'spec_helper'` line

Comment (askonev) [reply spec/convertion_by_format/check_open_html_by_screen_spec.rb]: Oh, thank you for the link) Next time I use google.

Comment (ShockwaveNN) [reply spec/convertion_by_format/check_open_html_by_screen_spec.rb]: @askonev Actually I've tried to use google to find this bug, but was not able to. But I remember that I reported this issue myself, so this allowed me to find it )

**Pull request merged**